CMakeLists.txt

```diff
@@ -18,6 +18,16 @@ OPTION(SPATH_LINK_STATIC "Default to static linking? (Needed for Cray)" OFF)
 
 # Find Packages & Files
 
+LIST(APPEND libspath_srcs
```

Comment:
I don't think you need any of the libspath_src lines in the top level CMakeLists.txt.  See if it still works if you delete these lines.